xcm-simulator/src/traits.rs

```diff
@@ -11,17 +11,17 @@ pub trait UmpMsgHandler {
 }
 
 pub trait HrmpMsgHandler {
-	fn handle_hrmp_msg(from: u32, msg: VersionedXcm) -> Result<(), ()>;
+	fn handle_hrmp_msg(from: u32, msg: VersionedXcm<()>) -> Result<(), ()>;
```

Comment:
Polkadot provides opaque types for `Xcm` and `VersionedXcm`, we can use them instead of adding the `()` to the generic ones. https://github.com/paritytech/polkadot/blob/761b4a07570cbbc0bebd763e4b5f66298b41b2bf/xcm/src/lib.rs#L43-L53

Reply:
Updated @shaunxw 🙏